torch/_inductor/codegen/triton.py

```diff
@@ -1023,7 +1023,22 @@ def codegen_header(self, code, no_x_dim=False):
                     f"{self.name} = {line}",
                 ]
             )
-        code.writeline(f"{x}mask = {self.name} < {x}numel")
+
+        max_block = config.triton.max_block[x.upper()]
+        no_mask_threshold = int(sympy.gcd(self.numel, max_block))
```

Comment:
better to see look at `self.numel % max_bock == 0`.

Reply:
That would give me a boolean, not a threshold that can be checked after the autotuner picks the block size.